core/iwasm/libraries/lib-wasi-threads/test/build.sh

```diff
@@ -9,6 +9,36 @@ set -eo pipefail
 CC=${CC:=/opt/wasi-sdk/bin/clang}
 WAMR_DIR=../../../../..
 
+rm -f *.aot
+rm -f *.wasm
+
+# Set default values
+enable_stress_tests=false
+
+# Function to display usage information
+function display_usage() {
+    echo "Usage: $0 [--enable_stress_tests]"
+    echo "Options:"
+    echo "  --enable_stress_tests   Enable stress tests"
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --enable_stress_tests)
+            enable_stress_tests=true
+            ;;
+        *)
+            display_usage
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+# Stress tests names (to include only in nightly build)
```

Comment:
why is that? Are they taking too long to be executed as part of regular builds?

Reply:
for now they are not but I was expecting that if we plan to add any new stress-tests later we won't include them into the regular build. 

How do you think, should this test now be a part of regular build?

Reply:
I'd keep them in the regular build until it's taking too long time and becomes a problem.